TestWebService/testhttp2.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# This source file is part of the Apodini open source project
+#
+# SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+#
+# SPDX-License-Identifier: MIT
+#
+
+set -e
+curl --fail --insecure --request GET --data-binary "@teststreamingrequest" -H "Content-Type: application/data" --http2-prior-knowledge --output /dev/null https://localhost/http/countdown
```

Comment:
Added this in a separate sh file because the Docker Compose & Traefik stuff is running via port 80, which I didn't want to mess with. swift-nio-http2 only supports HTTP/2 via HTTPS, so that's why I had to use HTTPS on port 443.